lib/qam.pm

```diff
@@ -16,10 +16,12 @@ use utils qw(zypper_call handle_screen zypper_repos);
 use JSON;
 use List::Util qw(max);
 use version_utils qw(is_sle is_transactional);
+use transactional qw(trup_call check_reboot_changes);
 
 our @EXPORT
   = qw(capture_state check_automounter is_patch_needed add_test_repositories disable_test_repositories enable_test_repositories
-  add_extra_customer_repositories ssh_add_test_repositories remove_test_repositories advance_installer_window get_patches check_patch_variables);
+  add_extra_customer_repositories ssh_add_test_repositories remove_test_repositories advance_installer_window get_patches
+  check_patch_variables install_package);
 use constant ZYPPER_PACKAGE_COL => 1;
 use constant OLD_ZYPPER_STATUS_COL => 4;
 use constant ZYPPER_STATUS_COL => 5;
```

Comment:
I think you pushed the wrong code xD, let's have all this in lib/package_manager.pm? or lib/pkgmgr.pm

Reply:
code isright

Reply:
It's in `package_utils` now

Reply:
@mdoucha lets leave its own package, likely we'll need much more in the near future